framework/behaviors/SluggableBehavior.php

```diff
@@ -239,7 +241,7 @@ protected function validateSlug($slug)
     /**
      * Generates slug using configured callback or increment of iteration.
      * @param string $baseSlug base slug value
-     * @param int $iteration iteration number
+     * @param integer $iteration iteration number
```

Comment:
This is wrong change. Should be `int` because `integer` can't be used as return type in PHP 7+.